src/NHSD.BuyingCatalogue.Solutions.API/Controllers/RoadmapController.cs

```diff
@@ -50,6 +50,6 @@ public async Task<ActionResult> Get([Required] string id)
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         public async Task<ActionResult> Update([Required] string id, UpdateRoadmapViewModel model) =>
-            (await mediator.Send(new UpdateRoadmapCommand(id, model?.Summary))).ToActionResult();
+            (await mediator.Send(new UpdateRoadMapCommand(id, model?.Summary))).ToActionResult();
```

Comment:
UpdateRoadmapViewModel should be updated to provide consistency here

Reply:
I think both Oxford and Cambridge dictionaries define road map like that (as two discrete words). I think the only place I've seen it as a single word is in BAPI! 😃 I'll update the 'view model' – that naming needs fixing too but that can come later – class as well to be consistent.